### DISCUSSION TITLE
Game Crash

### DISCUSSION THREAD

**Discussion by fras2560** [tsunamods-codes/7th-Heaven#225] [Q&A] [answered]
The game seems to play fine for a bit and then crash. I found the crash report. Is there a way to figure out why it is crashing?

**Comment by fras2560** [comment]
It seems to be when switching from one area to another. 
![image](https://github.com/user-attachments/assets/3e9e4ce7-388e-4cab-b2f7-ae80fef22721)

**Comment by julianxhokaxhiu** [comment, marked as answer]
Please use Discord support channels which is a way faster approach. Thank you!

**Comment by fras2560** [comment]
@julianxhokaxhiu do you have a link to the discord channel?

**Reply by julianxhokaxhiu** [reply]
You can find that here: https://7thheaven.rocks/#download7h
